portfolio/src/main/webapp/js/comments.js

```diff
@@ -175,37 +189,54 @@ function postComment(event) {
   event.preventDefault();
 
   const commentForm = document.getElementById("comment-form");
+  const commentPostButton = commentForm.elements["submit-button"];
 
   // Form data is encoded strictly in the form multipart/form-data
-  // However, Java servlet only supports application/x-www-form-urlencoded
-  // So use URLSearchParams to convert the data to the supported format
-  const formData = new URLSearchParams(new FormData(commentForm)).toString();
+  // which is the only supported format for submission to blobstore.
+  // If blobstore is no longer used, should instead convert to 
+  // application/x-www-form-urlencoded, which is the only supported format 
+  // when sending directly to a Java servlet. 
+
+  // Sending with application/x-www-form-urlencoded can be done like this:
+  // const formData = new URLSearchParams(new FormData(commentForm)).toString();
+  // const requestOptions = {
+  //   method: 'POST',
+  //   headers: {
+  //     'Content-Type': 'application/x-www-form-urlencoded'
+  //   },
+  //   body: formData
+  // };
 
   const requestOptions = {
     method: 'POST',
-    headers: {
-      'Content-Type': 'application/x-www-form-urlencoded'
-    },
-    body: formData
+    body: new FormData(commentForm)
```

Comment:
```suggestion
    body: new FormData(commentForm),
```
nit

Reply:
Oh wow, good eye! Didn't realize that was a [style requirement](https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma) until now!

Reply:
Ah yeah honestly I forgot that part, though now I remember reading it. I just like having the commas there so future diffs are cleaner :)